### DISCUSSION TITLE
Support for Tailwind UI libraries, e.g. preline, flowbite, etc?

### DISCUSSION THREAD

**Discussion by boosh** [jrief/django-formset#114] [Q&A]
Hi, this looks great, thanks. 

I just have one question that's not immediately clear. Would it be easy to use this library with UI libraries built on Tailwind e.g. preline or flowbite? I guess the styling could be achieved by creating new templates, but is the JS in those packages likely to conflict with the django-formset web component? 

I think ideally I'd like to use preline as much as possible but with django-formset for validation and integration with django. Would that be OK or likely to cause trouble?

Thanks

**Comment by jrief** [comment]
In don't have any experience with Preline, only Flowbite. Since web components are self contained, in my opinion they should work with JavaSripe from any external framework. I'm curios if you are able to get any of these to work and would be interested in ceating its own support branch.

**Reply by boosh** [reply]
OK thanks. Once I get round to styling my site I'll see if I can get them working together.
